cobalt/BUILD.gn

```diff
@@ -75,27 +77,17 @@ action("cobalt_build_info") {
   ]
 }
 
-static_library("common") {
-  sources = [
-    "user_agent_platform_info.cc",
-    "user_agent_platform_info.h",
-  ]
-
-  deps = [
-    ":cobalt_build_info",
-    "//base",
-    "//starboard/common",
-    "//v8:v8_version",
-  ]
-}
-
-target(gtest_target_type, "common_unittests") {
+test("cobalt_unittests") {
   testonly = true
 
-  sources = [ "user_agent_platform_info_test.cc" ]
+  sources = [
+    "//cobalt/user_agent/user_agent_platform_info_test.cc",
+    "//starboard/common/test_main.cc",
```

Comment:
Is there any reason this can't use `//base/test:run_all_unittests`?

Reply:
Oh thanks for pointing out, I didn't find this `//base/test:run_all_unittests` target.